.github/workflows/ci_linting.yml

```diff
@@ -90,3 +90,34 @@ jobs:
         with:
           clang-format-version: '15'
           include-regex: '^(\.\/)?(api|bin|crypto|stuffer|error|tls|utils|tests\/unit|tests\/testlib).*\.(c|h)$'
+  nixflake:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: nixbuild/nix-quick-install-action@v21
+        with:
+          nix_conf: experimental-features = nix-command flakes
+      - name: nix flake check
+        run: nix flake check
```

Comment:
I plan on merging a PR that is only tested on x86_64-linux. Ultimately the goal is to support all systems. Can we add: `NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1 nix flake check --impure` for the moment?

Similarly: we are currently depending on Python3.9. The `pyopenssl` package is a test depend that is marked broken in the nix package repo. The goal is to bump our python dependency (`pyopenssl` for Python3.10 is not marked broken). Can we add `NIX_PKGS_ALLOW_BROKEN=1` for the moment?